47852-0.txt

```diff
@@ -815,7 +815,7 @@ into the cauldron. From the moment that I perceived we were actually
 behind the cataract, and not in a mere cloud of spray, the enjoyment was
 intense. I not only saw the watery curtain before me like tempest-driven
 snow, but, by momentary glances, could see the crystal roof of this most
-wonderful of Nature’s palaces. The precise point where the flood quitted
+wonderful of Nature’s palaces. The precise point where the flood quit

```

Comment:
"quitted" is old-fashioned, not an error:
https://books.google.com/ngrams/graph?content=quit+%2C+quitted&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cquit%3B%2Cc0%3B.t1%3B%2Cquitted%3B%2Cc0